src/main/java/org/quantumbadger/redreader/activities/InboxListingActivity.java

```diff
@@ -342,6 +354,19 @@ public void run() {
 						this);
 
 				return true;
+			case OPTIONS_MENU_SHOW_UNREAD_ONLY:
+				if (!item.isChecked()) {
+					item.setChecked(true);
+					editor.putBoolean("onlyUnread", true);
```

Comment:
This string (`onlyUnread`) should be in `strings.xml` and ideally should have a wrapper in `PrefsUtility.java`. See this for an example:

https://github.com/QuantumBadger/RedReader/blob/master/src/main/java/org/quantumbadger/redreader/common/PrefsUtility.java#L160